BullsAndCows/SecretGenerator.cs

```diff
@@ -6,7 +6,24 @@ public class SecretGenerator
     {
         public virtual string GenerateSecret()
         {
-            throw new NotImplementedException();
+            string randomString = GenerateInt().ToString();
+            while (randomString.Length < 4)
+            {
+                string newNumber = GenerateInt().ToString();
+                if (!randomString.Contains(newNumber))
+                {
+                    randomString += newNumber;
+                }
+            }
+
+            Console.WriteLine(randomString);
```

Comment:
look like we do not need to display on the console

Reply:
just for check generator, I deleted later but forget to commit the last time